tensorboard/webapp/metrics/data_source/metrics_data_source.ts

```diff
@@ -37,9 +39,21 @@ import {
   TagToRunSampledInfo,
   TimeSeriesRequest,
   TimeSeriesResponse,
+  PersistableSettings,
 } from './types';
 
 const HTTP_PATH_PREFIX = 'data/plugin/timeseries';
+const LOCAL_STORAGE_KEY = '_tb_global_settings';
```

Comment:
Given the stated concern about coupling, should we namespace the key so that it is specific to timeseries?  e.g. `LOCAL_STORAGE_KEY = '_tb_global_settings.timeseries'` so that in the future, another feature's setting named `ignoreOutliers` with a very different meaning does not collide.  Alternatively, we could make the setting names specific enough that they should never collide e.g. "timeseriesScalarSmoothing", "timeseriesTooltipSort".